docs/recipes/custom-transformer.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Custom transformer
+---
+
+## Use case
+
+Sometimes the default response serializer isn't what you need and you're looking for a way to overwrite the default serialization for a response.
+
+For example, the default `JSON.stringify` implementation doesn't serialize `BigInt` values correctly, so you need to overwrite it with another serializer.
+
+## Defining a transformer
+
+Use the `createResponseComposition` API to create a custom transformer by defining a transformer.
+Assign the transformer to the `defaultTransformers` property, which can take multiple transformers.
```

Comment:
It would be useful to reference the [Response transformer](https://mswjs.io/docs/basics/response-transformer) page at least once during this description, so developers could navigate and learn more what that means.